### PR TITLE
8940iu content updates

### PR DESCRIPTION
**Pull request by bshyong**
## Description
Adds content updates

## Testing done
Local testing (content changes only)

## Acceptance criteria
- [x] Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/18458

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/content/unemployabilityDates.jsx [hunk 1]
```diff
@@ -29,18 +28,18 @@ export const dateFieldsDescription = (
...
+        This is the date you could no longer work full time or part time due to
         your service-connected disability.
       </p>
       <h5>Date you last worked full-time</h5>
       <p>
-        This is the date you could no longer work full-time due to your
+        This is the date you could no longer work full time due to your
         service-connected disability.
       </p>
       <h5>Date your disability began to affect your full-time employment</h5>
```
Comment (cvalarida) [inline src/applications/disability-benefits/all-claims/content/unemployabilityDates.jsx]: @peggygannon Should this `full-time` also be `full time`?

**Review by cvalarida — APPROVED** [review]
:+1:

**Comment by peggygannon** [comment]
Hi, @bshyong I want to confirm that the edits were implemented? I'm looking on staging and the first few edits aren't there. I just want to check before I continue on: 

https://staging.va.gov/disability/file-disability-claim-form-21-526ez/unemployability-walkthrough-choice

- Header still needs to be updated TO: Send former employers a request for information form

- change radio button to read:
I already filled out VA Form 21-8940 in paper form and want to upload it.  

https://staging.va.gov/disability/file-disability-claim-form-21-526ez/unemployability-disabilities

- Update subhead to read:
Rated, service-connected, and new disabilities

- In the second paragraph, change “your” to “the” / delete the extra “disabilities” / add “and keeping” like this TO:
Below are the rated, service-connected disabilities, and new disabilities you’re claiming. Please choose the disability or disabilities that prevent you from getting and keeping a steady job (substantially gainful employment).

File: src/applications/disability-benefits/all-claims/content/unemployabilityDates.jsx (hunk 1 above)
Comment (bshyong) [reply src/applications/disability-benefits/all-claims/content/unemployabilityDates.jsx]: @peggygannon Let me know if these should be consistent as Chris mentioned; if so I'll make the change and we can merge to staging for visual review

**Comment by cvalarida** [comment]
@peggygannon Until this gets merged in, the changes won't be in staging.

**Review by peggygannon — APPROVED** [review]
@bshyong Thank you. The edits look good.

**Pull request merged**